src/System.Data.SqlClient/tests/Tools/TDS/TDS.EndPoint/Cryptography/CryptKeyProviderInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.SqlServer.TDS.EndPoint.Cryptography
+{
+    /// <summary>
+    /// Structure that contains provider information
+    /// </summary>
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct CryptKeyProviderInfo
```

Comment:
The Interop in Cryptography folder is not being used any where. I am going to delete the folder. 
I had removed the reference to the folder from the csproj file. But I had missed deleting this folder.